docs/reference/settings.yaml

```diff
@@ -1195,6 +1195,13 @@ settings:
       - Type: [base64 encoded] `string` or inline policy structure in config file
       - **Required** However, pomerium will safely start without a policy configured, but will be unable to authorize or proxy traffic until the configuration is updated to contain a policy.
     doc: |
+
+      ::: warning
+      The `policy` field as a top-level configuration key has been replaced with [`routes`](/reference.html#routes). Moving forward, define policies within each defined route.
+      
+      Existing policy definitions will currently behave as expected, but are deprecated and will be removed in a future version of Pomerium.
+      :::
+
```

Comment:
@travisgroth this warning is part of the discussion we had in Slack on how to move folks from policy to routes.